workspace/src/utility.js

```diff
@@ -14,11 +14,11 @@ export function makeOpDel (pos, length) {
   };
 };
 
-export function makeOpMov (pos, value, des) {
+export function makeOpMov (pos, length, des) {
   return {
     type : "mov",
     pos : pos,
-    value : value,
+    value : length,
```

Comment:
I think it should be value: value.

@jam-world in `es6`, you do `{value}` and it is the same as `value: value`.